source/developers/transforms-with-object-lambda.rst

```diff
@@ -0,0 +1,353 @@
+- :doc:`The MinIO Go SDK </developers/go/minio-go>`
+
+
+Create a Handler
+~~~~~~~~~~~~~~~~
+
+The sample handler, written in Python, retrieves the target object using a `presigned URL <https://min.io/docs/minio/linux/developers/go/API.html#presigned-operations>`__ generated by the caller.
+The handler then transforms the object's contents and returns the new text.
+It uses the `Flask web framework <https://flask.palletsprojects.com/en/2.2.x/>`__ and Python 3.8+. 
+
+The following command installs Flask and other needed dependencies:
+
+.. code-block:: shell
+   :class: copyable
+
+   pip install flask requests
+
+The handler calls ``swapcase()`` to change the case of each letter in the original text.
+It then sends the results back to MinIO, which returns it to the caller.
+
+.. code-block:: py
+   :class: copyable
+
+   from flask import Flask, request, abort, make_response
+   import requests
+
+   app = Flask(__name__)
+   @app.route('/', methods=['POST'])
+   def get_webhook():
+      if request.method == 'POST':
+         # Get the request event from the 'POST' call
+         event = request.json
+
+	 # Get the object context
+         object_context = event["getObjectContext"]
+
+         # Get the presigned URL
+	 # Used to fetch the original object from MinIO
+         s3_url = object_context["inputS3Url"]
+
+         # Extract the route and request tokens from the input context
+         request_route = object_context["outputRoute"]
+         request_token = object_context["outputToken"]
+
+         # Get the original S3 object using the presigned URL
+         r = requests.get(s3_url)
+         original_object = r.content.decode('utf-8')
+
+         # Transform the text in the object by swapping the case of each char
+         transformed_object = original_object.swapcase()
+
+         # Return the object back to Object Lambda, with required headers
+         # This sends the transformed data to MinIO
+	 # and then to the user
+         resp = make_response(transformed_object, 200)
+         resp.headers['x-amz-request-route'] = request_route
+         resp.headers['x-amz-request-token'] = request_token
+         return resp
+
+      else:
+         abort(400)
+
+   if __name__ == '__main__':
+      app.run()
+
+
+Start the Handler
++++++++++++++++++
+
+Use the following command to start the handler in your local development environment:
+
+.. code-block:: shell
+   :class: copyable
+
+   python lambda_handler.py
+
+The output resembles the following:
+
+.. code-block:: shell
+
+    * Serving Flask app 'lambda_handler'
+    * Debug mode: off
+   WARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.
+    * Running on http://127.0.0.1:5000
+   Press CTRL+C to quit
+
+Once the handler is running, start MinIO with the :envvar:`MINIO_LAMBDA_WEBHOOK_ENABLE` and :envvar:`MINIO_LAMBDA_WEBHOOK_ENDPOINT` environment variables to register the function with MinIO.
```

Comment:
Maybe swap the order.
Set the environment variables, then start MinIO.

Reply:
For this purpose, it's the correct order because you are running in a local shell. Added another heading and additional text to clarify you are starting MinIO for testing, with these environment variables. Maybe that is easier to follow. Something like this:

> Start MinIO
> 
> (envvar detail)
> 
> The following command starts MinIO in your local development environment:
> 
> (code block)

Reply:
@djwfyi Here's the relevant section in the restaged version
http://192.241.195.202:9000/staging/DOCS-754/linux/html/developers/transforms-with-object-lambda.html#start-the-handler